portfolio/src/main/webapp/style.css

```diff
@@ -1,206 +1,201 @@
-    grid-template-columns: 1fr 4fr;
-    padding: 5px;
+  align-items: flex-start;
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  padding: 5px;
 }
 
 .navbar{
-    display: grid;
-    font-size: 20px;
-    grid-auto-flow: column;
+  display: grid;
+  font-size: 20px;
+  grid-auto-flow: column;
 }
 
 .nav-button{
-    background-color:inherit;
-    border:none;
-    color:inherit;
-    cursor:pointer;
-    display:inline-block;
-    overflow:hidden;
-    padding:8px 16px;
-    text-align:center;
-    text-decoration:none;
-    vertical-align:middle;
-    white-space:nowrap
+  background-color:inherit;
+  border:none;
+  color:inherit;
+  cursor:pointer;
+  display:inline-block;
+  overflow:hidden;
+  padding:8px 16px;
+  text-align:center;
+  text-decoration:none;
+  vertical-align:middle;
+  white-space:nowrap
 }
 
 .nav-left-side{
-    align-self: flex-start;
+  align-self: flex-start;
 }
 
 .navbar .navbar-item{
-    border:none;
-    display:block;
-    float:left;
-    outline:0s;
-    padding:8px 16px;
-    width:auto;
+  border:none;
+  display:block;
+  float:left;
+  outline:0s;
+  padding:8px 16px;
+  width:auto;
 }
 
 .nav-right-side{
-    justify-self: center;
-    margin-left: 150px;
+  justify-self: center;
+  margin-left: 150px;
 }
 
 .nav-right-side img{
-    width: 24px;
-    height: 24px;
+  width: 24px;
+  height: 24px;
 }
 
 .tooltip ~ #tooltipText{
-    background-color: rgba(255, 255, 255, 0.952);
-    border-radius: 6px;
-    color: rgba(11, 11, 165, 0.507);
-    height: 60px;
-    opacity: 1;
-    position: relative;
-    text-align: center;
-    top: 80px;
-    visibility: hidden;
-    width: 250px;
-    z-index: 100;
+  background-color: rgba(255, 255, 255, 0.952);
+  border-radius: 6px;
+  color: rgba(11, 11, 165, 0.507);
+  height: 60px;
+  opacity: 1;
+  position: relative;
+  text-align: center;
+  top: 80px;
+  visibility: hidden;
+  width: 250px;
+  z-index: 100;
 }
 
 .tooltip:hover ~ #tooltipText{
-    visibility: visible;
+  visibility: visible;
 }
 
 #user-comments{
-    display: none;
+  display: none;
 }
 
 
 .wrapper{
-    display: grid;
-    grid-template-rows: 2fr 2fr;
-    grid-template-columns: 1fr 1fr;
-    height: 450px;
-    justify-content: space-evenly;
-    width: 100%;
+  display: grid;
+  grid-template-rows: 2fr 2fr;
+  grid-template-columns: 1fr 1fr;
+  height: 450px;
+  justify-content: space-evenly;
+  width: 100%;
 }
 
 .wrapper .works-item a{
-    background:#F2F2F2;
-    color: black;
-    display: grid;
-    border-top-right-radius: 5px;
-    grid-template-rows:  1fr 3fr 1fr;
-    height: 500px;
-    margin-bottom: 30px;
-    margin-top: 10px;
-    opacity: 0.7;
-    position: relative;
-    top: 0;
-    transition: top ease 0.5s;
-    width: 80%;
-    text-decoration: none;
+  background:#F2F2F2;
+  color: black;
+  display: grid;
+  border-top-right-radius: 5px;
+  grid-template-rows:  1fr 3fr 1fr;
+  height: 500px;
+  margin-bottom: 30px;
+  margin-top: 10px;
```

Comment:
instead of having both, we prefer if you set this up as:
margin: 10px 0px 30px 0px;